src/client/sandbox/fetch.js

```diff
@@ -123,35 +93,52 @@ export default class FetchSandbox extends SandboxBase {
     attach (window) {
         super.attach(window, window.document);
 
+        if (!window.fetch)
```

Comment:
It is wrong.  A page can contain an own fetch implementation.
We should check this condition using the stored in `nativeMethods` value.

```js
if (!nativeMethods.fetch)
     return;
```